### PR TITLE
#253 Smoke test examples

### PR DESCRIPTION
**Pull request by DavideD**
Issue https://github.com/hibernate/hibernate-reactive/issues/253

Run the examples on MySQL and PostgreSQL.

### CODE REVIEW

File: example/src/main/resources/META-INF/persistence.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
              xsi:schemaLocation="http://java.sun.com/xml/ns/persistence http://java.sun.com/xml/ns/persistence/persistence_2_0.xsd"
              version="2.0">
 
-    <persistence-unit name="example">
+    <persistence-unit name="postgresqlExamplePU">
```
Comment (gavinking) [inline example/src/main/resources/META-INF/persistence.xml:6]: Very minor but I would prefer `"postgresql-example"`.

File: example/src/main/resources/META-INF/persistence.xml [hunk 2]
```diff
@@ -15,9 +15,39 @@
...
+
+            <!-- SQL statement logging -->
+            <property name="hibernate.format_sql" value="true"/>
+            <!--property name="hibernate.show_sql" value="true"/-->
+
+        </properties>
+
+    </persistence-unit>
+
+    <persistence-unit name="mysqlExamplePU">
```
Comment (gavinking) [inline example/src/main/resources/META-INF/persistence.xml:40]: Very minor but I would prefer `"mysql-example"`.

File: example/src/main/java/org/hibernate/example/reactive/MutinyMain.java [hunk 3]
```diff
@@ -15,13 +15,22 @@
...
 public class MutinyMain {
 
+	private static final String DEFAULT_PU = "postgresqlExamplePU";
+
+	// The first argument can be used to select a persistenceUnit.
+	// Check resources/META-INF/persistence.xml for available names.
+	public static String persitenceUnitName(String[] args) {
+		return args.length > 0 ? args[0] : DEFAULT_PU;
+	}
+
```
Comment (gavinking) [inline example/src/main/java/org/hibernate/example/reactive/MutinyMain.java:25]: I would prefer to move all this stuff down to the bottom of the file so it's not the first thing people see.

Also there's no need to make a `private static final String DEFAULT_PU` for a string literal that is used in exactly one place.

**Comment by gavinking** [comment]
@DavideD I left a couple of small comments.

**Comment by DavideD** [comment]
Fixes #253 

Applied the changes and merged

Thanks

**Pull request merged**